docs/devices_api.yml

```diff
@@ -1,14 +1,14 @@
 swagger: '2.0'
 info:
   title: Deployments
-  version: '0.1'
+  version: '1'
   description: |
     An API for device firmware deployments. Intended for use by devices.
 
     Devices can get new updates and send information about current deployment status.
 
-host: 'docker.mender.io:8080'
-basePath: '/api/devices/0.1/deployments'
+host: 'docker.mender.io'
```

Comment:
BTW. maybe we should not provide the host at all, since it makes little sense. If I start the service locally, there's no way I will access it by going to http://docker.mender.io.